src/app/pages/questions/components/question/info-screen/info-screen.component.ts

```diff
@@ -49,10 +50,8 @@ export class InfoScreenComponent implements OnInit, OnChanges {
 
   initSections() {
     this.sections.map((item, i) => {
-      console.log(item.label)
-      if (item.label.includes('THINC-it')) {
-        this.isThincItReminder = true
-      }
+      if (item.label.includes('THINC-it'))
+        this.image = 'assets/imgs/thincIt_app_icon.png'
```

Comment:
as discussed previously, this should be updated to be added via a `url` (hosted on public domain -- could possibly use github) in the protocols so as to make it configurable and also remove hard coding for THINC IT.
Possible feature request for furture PRs.